src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -76,16 +76,37 @@
         (rr/status ((response :outcome) outcomes->codes))
         (rr/content-type "text/plain"))))
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;;; Web app
-
 (defn try-to-parse
   [body]
   (try
     (cheshire/parse-stream (io/reader body) true)
     (catch Exception e
       (log/debug e))))
 
+(schema/defn handle-cert-clean
+  [{:keys [body]}
+   ca-settings :- ca/CaSettings]
+  (let [{:keys [certnames async]} (try-to-parse body)
```

Comment:
.... We probably want to validate that async, if passed, is not true.

Reply:
Okay.

Reply:
Implicit there is that I would understand if we said, w/o the option we assume sync behavior and reserve the async field for the future if time scales require this to no longer be synchronous.

Reply:
That is how I was thinking about this, yes.